torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1319,9 +1373,19 @@ Error PyTorchModelLoader::loadSub(const torch::jit::Node *ptNode) {
                     glow::strFormat("Scalar must have value equal 1."));
 
   glow::NodeValue lhs;
-  ASSIGN_VALUE_OR_RETURN_ERR(lhs, getGlowNodeValueForValue(inputs[0]));
   glow::NodeValue rhs;
-  ASSIGN_VALUE_OR_RETURN_ERR(rhs, getGlowNodeValueForValue(inputs[1]));
+
+  if (hasGlowNodeValueForValue(inputs[0])) {
+    ASSIGN_VALUE_OR_RETURN_ERR(lhs, getGlowNodeValueForValue(inputs[0]));
+    ASSIGN_VALUE_OR_RETURN_ERR(
+        rhs, loadNodeValueOrBroadcastedConstant(inputs[1], lhs.dims()));
+  } else if (hasGlowNodeValueForValue(inputs[1])) {
+    ASSIGN_VALUE_OR_RETURN_ERR(rhs, getGlowNodeValueForValue(inputs[1]));
+    ASSIGN_VALUE_OR_RETURN_ERR(
+        lhs, loadNodeValueOrBroadcastedConstant(inputs[0], rhs.dims()));
+  } else {
+    return MAKE_ERR("Either lhs or rhs of arithmetic node must be a tensor");
+  }
```

Comment:
Are these code all replicated?

Reply:
Yeah, idk I could factor this part out. Probably we should just factor everything out of the arithmetic ops since they're very similar